gulp-config.js

```diff
@@ -1,19 +0,0 @@
-module.exports = function() {
```

Comment:
I think if it doesn't exist we create it. Saves us from having a placeholder in a folder.
In the future, if/when Commerce publishes .XA to NuGet we will easily be able to remove the task.